.github/workflows/release.yaml

```diff
@@ -7,43 +7,42 @@ concurrency:
 jobs:
   release:
     if: (github.event_name == 'create' && github.event.ref_type == 'tag') || github.event_name == 'merge_group'
-    name: Release
+    name: Release ${{ matrix.toolchain }} ${{ matrix.platform.target }}
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+        - target: x86_64-unknown-linux-gnu
+          command: build
+          args: --all --release --features mshv
+        - target: x86_64-unknown-linux-musl
+          command: build
+          args: --all --release --features mshv
+        - target: aarch64-unknown-linux-musl
+          command: build
+          args:  --all --release
+        toolchain:
+          - 1.74
+          #- stable
+          #- beta
+          #- nightly
```

Comment:
We should also remove the `toolchain` from the matrix list, and directly specify toolchain version say `1.74` for now in the steps, or make it as a part of the `matrix.platform` if you prefer to have it in a central place.

Reply:
done, removed